snuba/cli/__init__.py

```diff
@@ -17,8 +32,15 @@ def main() -> None:
     `OoO'  o   O `OoO'o `OoO' `OoO'o"""
 
 
-for loader, module_name, is_pkg in walk_packages(__path__, __name__ + "."):
-    module = __import__(module_name, globals(), locals(), ["__name__"])
-    cmd = getattr(module, module_name.rsplit(".", 1)[-1])
-    if isinstance(cmd, click.Command):
-        main.add_command(cmd)
+with sentry_sdk.start_transaction(op="snuba_init", name="Snuba CLI Initialization"):
+    for loader, module_name, is_pkg in walk_packages(__path__, __name__ + "."):
+        logger.info(f"Loading module {module_name}")
```

Comment:
It seems this will be really really verbose. Are you sure about logging this in `INFO`? It will show up in google cloud log.I feel it is not a very useful piece of info. I would demote this to DEBUG